### PR TITLE
Add data.json file for generating JSON representation

### LINKED ISSUE #931: JSON representation
Great work aggregating all this data!

I'm looking to build a Chrome extension that uses the information here -- it would be useful to have a JSON representation of the data.

Would you be open to a pull request to add this?

### PR DESCRIPTION
**Pull request by zbuc**
This closes #931 

If someone wants to figure out a better way of avoiding trailing commas than appending "": {}, go for it.

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+                        {% for website in section_file.websites %}
+                                "{{ website.name }}": {
+                                            "tfa": {% if website.tfa %}true{% else %}false{% endif %},
+                                            "url": "{{ website.url }}",
+                                            "img": "{{ website.img }}",
+                                            "twitter": "{{ website.twitter }}",
+                                            "sms": {% if website.sms %}true{% else %}false{% endif %},
+                                            "software": {% if website.software %}true{% else %}false{% endif %},
+                                            "phone": {% if website.phone %}true{% else %}false{% endif %},
+                                            "hardware": {% if website.hardware %}true{% else %}false{% endif %},
```
Comment (smholloway) [inline data.json]: I think we can kill the comma on the end here. Chrome loads it properly, but the JSON spec does not approve of trailing commas in JSON objects.

**Comment by smholloway** [comment]
Great contribution! It looks good locally.

**Comment by jdavis** [comment]
Cool. I can't test it right now but if it isn't merged by the time I get home, I'll do so. I'm excited for the extension; be sure to keep us in the loop on how it is going.

**Comment by jdavis** [comment]
Awesome. I think this looks great. The only thing that I think I will change is to include the full path for the image. It currently just says `{{website.img}}` but should be `/img/{{section.id}}/{{website.img}}`.

**Comment by jdavis** [comment]
You should be able to view it all here: https://twofactorauth.org/data.json

Let me know if you see/have any issues.

**Pull request merged**